### PR TITLE
NIAD-759 Documentation to support testing of object store

### PR DESCRIPTION
**Pull request by mattd-kainos**
* Enhanced documentation about how to run integration tests with external services in-the-loop

* Added a default mongo URI for intTest

### CODE REVIEW

**Review by michael-kainos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -69,32 +69,67 @@ The project includes mock interactions of external APIs (GPC, SDS) implemented i
 
 The folder `docker/wiremock/stubs` describes the supported interactions.
 
-## Unit test section 
+## How to run tests
+
+**Warning**: Gradle uses a [Build Cache](https://docs.gradle.org/current/userguide/build_cache.html) to re-use compile and
+test outputs for faster builds. To re-run passing tests without making any code changes you must first run 
+`./gradle clean` to clear the build cache. Otherwise, gradle uses the cached outputs from a previous test execution to
```
Comment (Broom94) [inline README.md:76]: should this be gradle wrapper?

File: README.md [hunk 2]
```diff
@@ -69,32 +69,67 @@ The project includes mock interactions of external APIs (GPC, SDS) implemented i
...
 
-## How to run all checks:
-* Navigate to `service`
-* Run: `./gradlew check`
+3. Run the integration tests
 
-## How to run e2e tests:
-* `docker-compose -f docker-compose-integration-tests.yml build && docker-compose -f docker-compose-integration-tests.yml up --exit-code-from integration_tests`
+    ```shell script
+    cd ../service
```
Comment (Broom94) [inline README.md:130]: ./service?

File: README.md (hunk 1 above)
Comment (Broom94) [reply README.md:76]: should we add navigate to service?

Comment (mattd-kainos) [reply README.md]: fixed typo, added a sentence about directory for gradle commands

File: README.md (hunk 2 above)
Comment (mattd-kainos) [reply README.md:129]: The users is already in `docker/` if they followed the previous step

**Review by Broom94 — APPROVED** [review]

**Review by Broom94 — DISMISSED** [review]

**Review by michael-kainos — APPROVED** [review]

**Review by Broom94 — APPROVED** [review]

**Pull request merged**